packages/vue/src/examples/pages/cart/Cart.vue

```diff
@@ -0,0 +1,464 @@
+        >
+          {{ totalItems }}
+        </SfCircleIcon>
+      </template>
+      <div v-if="totalItems > 0" class="my-cart">
+        <div class="my-cart__header">
+          <div class="my-cart__heading">
+            <SfHeading
+              title="My cart"
+              class="sf-heading--no-underline sf-heading--left"
+            />
+            <button class="my-cart__close" @click="isCartSidebarOpen = false">
+              <SfIcon icon="cross" size="20px" color="#BEBFC4" />
+            </button>
+          </div>
+          <h3 class="my-cart__total-heading">Total items</h3>
+        </div>
+        <div class="my-cart__product-list">
+          <SfCollectedProduct
+            v-for="(product, key) in products"
+            :key="key"
+            :image="product.image"
+            :title="product.title"
+            :regular-price="product.price.regular | price"
+            :special-price="product.price.special | price"
+            :stock="product.stock"
+            v-model="product.qty"
+            @click:remove="removeHandler(product)"
+            class="product"
+          >
+            <template #configuration>
+              <div class="product__properties">
+                <SfProperty
+                  v-for="(property, key) in product.configuration"
+                  :key="key"
+                  :name="property.name"
+                  :value="property.value"
+                />
+              </div>
+            </template>
+            <template #actions>
+              <div class="product__actions">
+                <div style="text-decoration: underline; line-height: 1.6">
+                  Save for later
+                </div>
+                <div style="text-decoration: underline; line-height: 1.6">
+                  Add to compare
+                </div>
+              </div>
+            </template>
+          </SfCollectedProduct>
+        </div>
+        <div class="my-cart__summary">
+          <SfProperty class="my-cart__total sf-property--full-width">
+            <template #name>
+              <span class="sf-property__name">TOTAL</span>
+            </template>
+            <template #value>
+              <SfPrice :regular="totalPrice | price" class="sf-price--big" />
+            </template>
+          </SfProperty>
+          <SfButton class="sf-button--full-width">Go to checkout</SfButton>
+        </div>
+      </div>
+      <div v-else class="my-cart">
+        <div class="my-cart__header">
+          <div class="my-cart__heading">
+            <SfHeading
+              title="My cart"
+              class="sf-heading--no-underline sf-heading--left"
+            />
+            <button class="my-cart__close" @click="isCartSidebarOpen = false">
+              <SfIcon icon="cross" size="20px" color="#BEBFC4" />
+            </button>
+          </div>
+        </div>
+        <div class="empty-cart">
+          <svg class="empty-cart__icon">
```

Comment:
don't we have it in icons.js? if we don';t just add it ;)

Reply:
but it's too complicated icon :D it's most like the image (it have mask, group, path, circle..)

Reply:
Ah you're right. Maybe just ezternwlize it to svg file then (in shared package). Keeping whole svg code here is not very clean